### DISCUSSION TITLE
Server lists not filling with data after new ISP activation

### DISCUSSION THREAD

**Discussion by Zyfuss** [jamulussoftware/jamulus#2898] [Technical Support & Help]
After moving to a new ISP (SKY WIFI that I guess is not only in Italy but meybe Germany and UK) the the lists of the servers don't fill with data. Precisely the All Genre 1 and 2 not fill up, All Genre 3, Rock, Jazz fill all the columns while Classica and Corale fill only the current number of Players column (No max players and Location).
Also I notice that at first opening of the list all the ping times are > 500 than at first refresh all seems ok.
The connection is OK and through ETH cable, before the ISP changing all was absolutely wonderful.
From some googling I got that the problem could be a packet fragmentation issue an could be managed changing the MTU parameter on the router. BUT, the WEB management interface on the router is very essential and is not possible to tweak any paramater.
This situation is a pain in the a.s... Did anyone experience similar problems and did you fixed them?
Thank you for any support.... 
Ciao.

**Comment by pljones** [comment]
I think the general fix is "Get a better ISP"...

Changing the underlying protocol on handling the lists is on the "things we're thinking about doing (but haven't actually started looking at seriously)" list.
